internal/login.go

```diff
@@ -46,7 +46,7 @@ func Login(loginUrl string, targetHost string, targetPort int) (string, error) {
 	r.HandleFunc("/", func(w http.ResponseWriter, r *http.Request) {
 		// try and extract access token from headers
 		accessToken = r.Header.Get("access_token")
-		s.Close()
+		_ = s.Close()
```

Comment:
Even though it's idiomatic to ignore errors on close, it might be good to log a warning if it fails.